internal/handlers/handler_firstfactor.go

```diff
@@ -104,6 +105,10 @@ func FirstFactorPost(ctx *middlewares.AutheliaCtx) {
 	userSession.AuthenticationLevel = authentication.OneFactor
 	userSession.LastActivity = time.Now().Unix()
 	userSession.KeepMeLoggedIn = keepMeLoggedIn
+	refresh, refreshInterval := ctx.Providers.UserProvider.GetRefreshSettings()
+	if refresh {
+		userSession.RefreshTTL = ctx.Clock.Now().Add(refreshInterval)
```

Comment:
I set mine to 1 month for testing, and the intention was for it to not be affected by session tuning at all. The tests me and @nightah did confirmed the behavior. I will look at making sure a mock checks this.

Reply:
So regarding sessions. What occurs is because the default for time.Time is the equivalent of -62135596800 unix epoch time, it will be that value if nothing is provided by session storage. So unless the server clock is slightly out of sync, they should be fine.

Note: for anyone reading this, -62135596800 is the 1st of January 0001, 09:39:52. Not sure why go chose this time, but interesting trivia!